### ISSUE TITLE
Request for enhancement: support data sets

### ISSUE THREAD

**Issue by Cris70** [zowe/zowe-explorer-ftp-extension#10]
Dear developers, I have recently installed the Zowe Explorer FTP extension, hoping to be able to work with z/OS datasets.
Unfortunately I discovered that it only enables me to work with USS files. While this is very convenient, I could have reached the same goal using one of the other FTP extensions that are available for VSCode.
The real value would be, for me, being able to work seamlessly with z/OS datasets. I see that this is possible using Zowe-cli from the command line with a zftp profile. Would it be possible to add such functionality in the Zowe Explorer FTP Extensions?
Thank you in advance
Cris

**Comment by DavidMunera7** [comment]
I completely agree with Cris70. Once I installed the Zowe Explorer FTP extension, I was somewhat let down. I wanted to work as well with z/OS datasets. If the developers could make something happened like Cris70 mentioned that would be awesome.
Thanks
David M.
